Fabric.Identity.API/Documentation/SecurityRequirementsOperationFilter.cs

```diff
@@ -19,13 +19,24 @@ public SecurityRequirementsOperationFilter(IOptions<AuthorizationOptions> author
 
         public void Apply(Operation operation, OperationFilterContext context)
         {
-            var controllerPolicies = context.ApiDescription.ControllerAttributes()
-                .OfType<AuthorizeAttribute>()
-                .Select(attr => attr.Policy);
-            var actionPolicies = context.ApiDescription.ActionAttributes()
-                .OfType<AuthorizeAttribute>()
-                .Select(attr => attr.Policy);
-            var policies = controllerPolicies.Union(actionPolicies).Distinct();
+            List<string> policies = new List<string>();
+
+            if (context.ApiDescription.ActionDescriptor is
```

Comment:
It looks like you could get the policy names that determines access to the resource, by consolidating to this and add a .Select(attr => attr.Policy) at the end.         
       var authAttributes = context.MethodInfo.DeclaringType.GetCustomAttributes(true)
            .Union(context.MethodInfo.GetCustomAttributes(true))
            .OfType&lt;AuthorizeAttribute&gt;(); 
https://github.com/domaindrivendev/Swashbuckle.AspNetCore#operation-filters